pages/packages.md

```diff
@@ -39,4 +39,28 @@ Some advanced package creation tools include:
 - [fpm](https://github.com/jordansissel/fpm), a third-party tool that allows users to quickly and easily make a variety of packages (including RPM and DEB packages)
 - [PackPack](https://github.com/packpack/packpack), a simple tool to build RPM and Debian packages from git repositories.
 
-Learn more about Buildkite Packages in the [Getting started](/docs/packages/getting-started) guide.
+## Buildkite Packages
+
+Buildkite Packages provides registries for your [packages](/docs/packages#an-introduction-to-packages) and other package-like formats such as container images and Terraform modules. As well as holding a collection of packages, each registry may also contain metadata describing a variety of attributes for these packages such as, package versions, supported operating system versions and processor architecture, dependencies, and so on.
+
+### Supported package ecosystems
+
+Currently, Buildkite Packages supports the following package ecosystems:
+
+- Alpine
+- Docker container images
+- deb (Debian and Ubuntu)
+- gem (RubyGems)
+- Java (Maven and Gradle)
+- Node.js (npm)
+- Python (PyPI)
+- rpm (Fedora and RHEL)
+- Terraform
```

Comment:
Might be worth adding modules here, because we don't support providers (yet)

```suggestion
- Terraform (modules)
```

Reply:
Thanks Sam :-)

Reply:
TheNewFactory@Art&Contents Jcarlossantosferreiros77@gmail.com
Personal equipos proyectos si quieres formar parte  unete